base/uk.ac.stfc.isis.ibex.opis/resources/sans2d_vacuum_bafdet_indicator.py

```diff
@@ -0,0 +1,16 @@
+from org.csstudio.opibuilder.scriptUtil import PVUtil
+
+bafdet_location_z_pv = pvs[0]
+bafdet_location_z = PVUtil.getDouble(pvs[0])
+min_x = 150.0
+max_x = 800.0 + 150.0
+diff_x = max_x - min_x
+min_z = 0.0
+max_z = 12500.0
+diff_z = max_z - min_z
+sizing_value = diff_x / diff_z
+size_offset = 40.0 / 2
+widget_new_x = (bafdet_location_z * sizing_value) + min_x - size_offset
+
+widget.setPropertyValue("x", round(widget_new_x, 1))
+widget.setPropertyValue("tooltip", str(bafdet_location_z_pv) + "\n" + str(widget_new_x))
```

Comment:
I think I meant to display the actual position of the baffle/detector, but I think saying what FD means is better.

The scripts only run when the setpoint changes, so that shouldn't be an issue. However, I have tried it out with using the baffles actual position and because these are changing often it does slow the OPI down. I think moving the calculation to the FINS_VAC IOC might make sense?